### DISCUSSION TITLE
Getting the selected sequence string

### DISCUSSION THREAD

**Discussion by Marwa-Da** [molstar/molstar#994] [Q&A] [answered]
Hi all,

I want to get the selected sequence string if I have their loci. Any hints on how to achieve this?

**Comment by dsehnal** [comment, marked as answer]
You can have a look at `model.sequence` and you can see how the UI is implemented: https://github.com/molstar/molstar/blob/e773824fb543f2a33989a63f7d0c96df499ec40f/src/mol-plugin-ui/sequence/sequence.tsx#L60 . That should have enough hints for you.
